### DISCUSSION TITLE
v1.9.3

### DISCUSSION THREAD

**Discussion by vran-dev** [vran-dev/PrettyZoo#206] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/vran-dev/PrettyZoo/releases/tag/v1.9.3'>v1.9.3</a>.</em>

**Comment by artheus** [comment]
@vran-dev The Mac version does not seem to be built properly.
I've downloaded it multiple times to test, every time I get the same error message:

![Screenshot 2021-08-25 at 12 50 33](https://user-images.githubusercontent.com/618946/130778137-62b41ad1-0940-47fe-9114-aa6f3af6724b.png)

EDIT: I also get the same error message with the previous release (v1.9.2)
EDIT 2: I created an issue report for this #219

**Reply by lizhenzhaofirst** [reply]
I also get the same error message with the release (v1.9.5 & v1.9.4)
